lms/djangoapps/ccx/api.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tasks related to the Custom Courses feature.
+"""
+from instructor.access import allow_access, list_with_level
+from instructor.enrollment import (
+    enroll_email,
+    get_email_params,
+)
+
+
+def add_master_course_staff_to_ccx(course, ccx_key, display_name):
+    """
+    Added staff role on ccx to all the staff members of master course. 
+    :param course: Master course instance
+    :param ccx_key: CCX course key
+    :param display_name: ccx display name for email
```

Comment:
Sorry to insist on this, but I think the way they want docstrings now is something like this:

```
Args:
    course (<the object type here>): Master course instance
    ccx_key (<the object type here>): CCX course key
    ...

Returns:
    None  # Not sure this is actually necessary
```